### PR TITLE
Fix before_snippet failing in release

### PR DESCRIPTION
**Pull request by bugadani**
Oops

### CODE REVIEW

**Review by MabezDev — APPROVED** [review]

**Comment by bugadani** [comment]
Sorry for the late push, one of the embassy comments was misbehaving

File: esp-hal-embassy/src/executor/thread.rs [hunk 1]
```diff
@@ -72,9 +72,7 @@ impl Executor {
         multi_core,
         doc = r#"
 
-    This will use software-interrupt 3 which isn't
-    available for anything else to wake the other core(s).
-    "#
+This will use software-interrupt 3 which isn't available for anything else to wake the other core(s)."#
```
Comment (bugadani) [inline esp-hal-embassy/src/executor/thread.rs:75]:
```
warning: could not parse code block as Rust code
  --> src\executor\thread.rs:70:5
   |
70 | /     /// Create a new Executor.
71 | |     #[cfg_attr(
72 | |         multi_core,
73 | |         doc = r#"
...  |
76 | |     available for anything else to wake the other core(s).
77 | |     "#
   | |______^
   |
   = note: error from rustc: prefix `isn` is unknown
   = note: `#[warn(rustdoc::invalid_rust_codeblocks)]` on by default
```

Comment (bjoernQ) [reply esp-hal-embassy/src/executor/thread.rs:75]: where did that show up for you? Don't think I've seen this before

Comment (bugadani) [reply esp-hal-embassy/src/executor/thread.rs:75]: I built the documentation for esp-hal-embassy, which we don't usually do.

**Comment by MabezDev** [comment]
So is this ready to go?

**Comment by bugadani** [comment]
I think so.

**Comment by nm17** [comment]
Feel lucky to have caught this error live :smile:

**Comment by nm17** [comment]
By the way, what is the ETA of this fix to crates.io?

**Pull request merged**